core/services/chainlink/config_general.go

```diff
@@ -345,6 +345,29 @@ func (g *generalConfig) StarkNetEnabled() bool {
 	return false
 }
 
+// TODO: Update this method once Aptos becomes public integration
+// BCI-3511: https://smartcontract-it.atlassian.net/browse/BCI-3511
+func (g *generalConfig) AptosEnabled() bool {
+	for _, c := range g.c.Aptos {
+		if c == nil {
+			continue
+		}
+		if v, ok := (*c)["Enabled"]; ok {
+			switch typedV := v.(type) {
+			case nil:
+				return true
+			case bool:
+				return typedV
+			case *bool:
+				return *typedV
+			default:
+				panic(fmt.Sprintf("AptosEnabled: 'Enabled' has unexpected type %T", v))
```

Comment:
Sorry I should have said more. If you add a `ValidateConfig` method to `RawConfig`, then you can return a user friendly error: https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/config/validate.go#L17

Reply:
thanks for this reference @jmank88 

Do you mean something like this?

```
// Validate returns an error if the Config is not valid for use, as-is.
func (c *RawConfig) ValidateConfig() error {
	if v, ok := (*c)["Enabled"]; ok {
		switch v.(type) {
		case nil, bool, *bool:
			return nil
		default:
			return fmt.Errorf("invalid type for 'Enabled': expected nil, bool, or *bool, got %T", v)
		}
	}

	return nil
}
```

And this `ValidateConfig()` would be called by the validate function of `generalConfig` [here](https://github.com/smartcontractkit/chainlink/blob/eace378772fe88db6c33f93b36d1ad2a421b339e/core/services/chainlink/config_general.go#L219)?